docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -153,56 +149,27 @@
       set_fact:
         config_permdir: "{{ platform_path + '/config/' + software_version }}"
 
-    # Check if DM already exists in either helmv2 or helmv3
-    - name: Get list of releases in helmv2
-      command: >-
-        /usr/local/sbin/helmv2-cli -- helm list --output json
-      environment:
-        KUBECONFIG: "/etc/kubernetes/admin.conf"
-      ignore_errors: yes
-      register: helmv2_releases
-
-    # Ansible returns error code 2 to helmv2_releases if helmv2-cli does not exist
-    - name: Check if helmv2-cli is installed
-      set_fact:
-        helmv2_installed: "{{ false if helmv2_releases.rc == 2 else true }}"
-
-    - name: Query for DM release in helmv2
-      shell: |
-        grep -iq '"name":"deployment-manager"' <<< {{ helmv2_releases.stdout_lines }}
-      ignore_errors: yes
-      register: helmv2_dm_exists
-      when: helmv2_installed
-
-    - name: Get list of releases in helmv3
-      command: >-
-        /usr/sbin/helm list --output json
-      environment:
-        KUBECONFIG: "/etc/kubernetes/admin.conf"
-      ignore_errors: yes
-      register: helmv3_releases
-
-    # Ansible returns error code 2 to helmv3_releases if helmv3 does not exist
-    - name: Check if helmv3 is installed
-      set_fact:
-        helmv3_installed: "{{ false if helmv3_releases.rc == 2 else true }}"
-
-    - name: Query for DM release in helmv3
-      shell: |
-        grep -iq '"name":"deployment-manager"' <<< {{ helmv3_releases.stdout_lines }}
-      ignore_errors: yes
-      register: helmv3_dm_exists
-      when: helmv3_installed
-
     - name: Check nfv-vim.dor_complete flag
       stat:
         path: /var/run/.nfv-vim.dor_complete
       register: dor_complete_file
 
+      # Check deployment-manager system app status
+    - name: Check deployment-manager app is status
+      shell: source /etc/platform/openrc; system application-list --nowrap | grep deployment-manager | awk '{ print $10 }'
+      register: dm_app_status
+
+    - name: Check deployment-manager pod is running
+      shell: |
+       kubectl -n platform-deployment-manager get pods | grep platform-deployment-manager- | awk 'NR == 1 { print $3 }'
+      environment:
+          KUBECONFIG: "/etc/kubernetes/admin.conf"
+      register: deployment_manager_pod_status
+
     - name: Set reconfiguration related facts
       set_fact:
-        reconfig_flag: "{{ true if (helmv2_installed and helmv2_dm_exists.rc == 0)
-                        or (helmv3_installed and helmv3_dm_exists.rc == 0)
+        reconfig_flag: "{{ true if (dm_app_status.stdout == 'applied' and
```

Comment:
expect to handle the case the dm_app_status.stdout != 'applied' or deployment_manager_pod_status.stdout != 'Running', we cannot simply set the reconfig flag = False in these cases